UI/js/factories/factories.js

```diff
@@ -119,4 +119,24 @@ dashboardApp.factory('Terms', function ($http) {
       );
     }
   };
+});
+
+
+/**
+ *  Factory to return  the Canvas/CTools todo data as promise
+ */
+
+dashboardApp.factory('ToDos', function ($http) {
+  return {
+    getToDos: function (url) {
+      return $http.get(url, {cache: true}).then(
+        function success(result) {
+          return prepareToDos(result);
+        },
+        function error() {
+          //console.log('errors');
```

Comment:
Why would you not want to see the errors in the console? Is this kept in so it can be turned on for testing?